pagopa/io-app
# chore(Bonus Pagamenti Digitali): [#176078094] Added supported credit cards page link

## List of changes proposed in this pull request
- Removed hardcoded supported card list
- Added link to external page

## How to test
Try to add a new card and see the link, this is the requested result from design:
![download](https://user-images.githubusercontent.com/26572302/101736686-18435900-3ac4-11eb-8aff-c1f22e2248c2.png)

The actual result is underlined being the default `<Link>` component
